### ISSUE TITLE
One action declaration missed in "mri.p4"

### ISSUE THREAD

**Issue by Wasdns** [p4lang/tutorials#28]
Hello,

I'm tring the demo named `mri` in the branch `P4D2`, and noticed that the table `swid` missed the declaration of `NoAction`. See [mri.p4/#L184](https://github.com/p4lang/tutorials/blob/p4d2/P4D2_2017/exercises/mri/solution/mri.p4#L184). It will raise the error like this:

```
Entering build directory.
Extracting package.
Reading package manifest.
> p4c-bm2-ss --p4v 16 "mri.p4" -o "mri.p4.json"
mri.p4(188): error: NoAction not present in action list
        default_action = NoAction();
                         ^^^^^^^^
Compile failed.
```

It should be like this:

```
    table swid {
        actions        = { add_swid; NoAction; }
        default_action =  NoAction();      
    }
```

By the way, I noticed that you are still working on the p4v1.2 specification and `p4c` compiler. Thank you for this great work!

Best regards,
Chen.

**Comment by jnfoster** [comment]
Thanks! 

(Minor point: the name of the language is P4<sub>16</sub>, and the version number released next week will be 1.0.0).

**Comment by Wasdns** [comment]
@jnfoster 

Thank you for reminding me of this matter. 👍
